templates/python.mk

```diff
@@ -26,12 +28,9 @@ COMPONENT_ARCHIVE_URL=
 COMPONENT_ARCHIVE_HASH=
 COMPONENT_LICENSE=
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/setup.py.mk
-include $(WS_MAKE_RULES)/ips.mk
+TEST_TARGET=$(NO_TESTS)
```

Comment:
Unless there aren't any python tests and will never be, shouldn't this also only be set if not yet defined?

Reply:
For python packages most are without tests so we could disable them by default.

Reply:
But then we should fix all the packages using already BUILD_STYLE if needed:
```
narval> ggrep -l "BUILD_STYLE" */Makefile | cut -f 1 -d '/'
argcomplete
argh
attrs
automat
barman
ccsm
constantly
hamcrest
hyperlink
hypothesis
incremental
kafka-python
netifaces
pillow
psutil
psycopg2
pycups
python-dateutil
python-memcached
python-xlib
pyxdg
redis-py
twisted
zope-interface

```

Reply:
Enabling test target by default is a good practice ;)

Reply:
OK I will change it then :)